lib/Runtime/Base/FunctionBody.h

```diff
@@ -3690,7 +3690,7 @@ namespace Js
         static uint const ScopeMetadataSlotIndex = 1;    // Either a FunctionBody* or DebuggerScope*
         static uint const FirstSlotIndex = 2;
     public:
-        ScopeSlots(Var* slotArray) : slotArray(slotArray)
+        ScopeSlots(Var* slotArray) : slotArray((Field(Var)*)slotArray)
```

Comment:
Field(Var)* is WriteBarrierPtr<T*> *,  Field(Var) is WriteBarrierPtr<T**>
weird the comment does not show the star after '>'

---
In reply to: [92464727](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92464727) [](ancestors = 92464727,92464304)